csharp/GrpcOpenTracing/GrpcOpenTracing/Handlers/InterceptedServerHandler.cs

```diff
@@ -1,80 +1,70 @@
-﻿using Grpc.Core;
-using GrpcOpenTracing.Propagation;
-using GrpcOpenTracing.Streaming;
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Grpc.Core;
+using Grpc.OpenTracing.Propagation;
+using Grpc.OpenTracing.Streaming;
 using OpenTracing;
 using OpenTracing.Propagation;
 using OpenTracing.Tag;
-using System;
-using System.Linq;
-using System.Threading.Tasks;
 
-namespace GrpcOpenTracing.Handlers
+namespace Grpc.OpenTracing.Handlers
 {
     internal class InterceptedServerHandler<TRequest, TResponse>
         where TRequest : class
         where TResponse : class
     {
-        private readonly GrpcTraceLogger<TRequest, TResponse> logger;
-        private readonly ServerCallContext context;
+        private readonly GrpcTraceLogger<TRequest, TResponse> _logger;
+        private readonly ServerCallContext _context;
 
         public InterceptedServerHandler(ITracer tracer, ServerCallContext context)
         {
-            this.context = context;
+            _context = context;
 
-            ISpan span = GetSpanFromContext(tracer);
-            this.logger = new GrpcTraceLogger<TRequest, TResponse>(span);
+            var span = GetSpanFromContext(tracer);
+            _logger = new GrpcTraceLogger<TRequest, TResponse>(span);
         }
 
         private ISpan GetSpanFromContext(ITracer tracer)
         {
-            return GetSpanFromHeaders(tracer, context.RequestHeaders, $"Server {this.context.Method}")
-                .SetTag(Tags.Component, Constants.TAGS_COMPONENT)
-                .SetTag(Tags.SpanKind, Tags.SpanKindServer)
-                .SetTag("peer.address", this.context.Peer)
-                .SetTag("grpc.method_name", this.context.Method)
-                .SetTag("grpc.headers", GetGrpcHeaders());
+            return GetSpanBuilderFromHeaders(tracer, _context.RequestHeaders, $"Server {_context.Method}")
+                .WithTag(Tags.Component, Constants.TAGS_COMPONENT)
+                .WithTag(Tags.SpanKind, Tags.SpanKindServer)
+                .WithTag("peer.address", _context.Peer)
```

Comment:
`peer.address` is defined in the [semantic conventions](https://github.com/opentracing/specification/blob/master/semantic_conventions.md) but not in `Tags`. Should I file an issue to add the missing ones, or are they maybe already in the latest version?